### ISSUE TITLE
Are you accepting PRs?

### ISSUE THREAD

**Issue by mjhea0** [humiaozuzu/awesome-flask#150]

**Comment by lt94** [comment]
@mjhea0  last commit was 8 months ago, it seems that the project was not maintained

**Comment by mjhea0** [comment]
Yeah, I just wish the maintainer would step down then since it's the most popular Awesome Flask. @humiaozuzu

**Comment by zachvalenta** [comment]
Or clarity about whether they will continue forward with the project

**Comment by zoidbergwill** [comment]
Have any of you folks tried e-mailing the maintainer / owner of the repo? Maybe if that doesn't work, we can make an "official"'ish fork under an awesome-flask org?

**Comment by zachvalenta** [comment]
@zoidbergwill good idea!

<img width="617" alt="screen shot 2019-01-27 at 9 41 23 am" src="https://user-images.githubusercontent.com/11388735/51802411-da165680-2217-11e9-90ce-627400c2a767.png">

**Comment by zachvalenta** [comment]
he's back :)

**Comment by mjhea0** [comment]
@zachvalenta is he?

**Comment by zachvalenta** [comment]
@mjhea0 yep! buncha outstanding PRs merged after we reached out
